gpflow/covariances/kufs.py

```diff
@@ -1,16 +1,22 @@
 import tensorflow as tf
-from ..inducing_variables import InducingPoints, Multiscale, InducingPatches
+from ..base import TensorLike
+from ..inducing_variables import InducingVariables, InducingPoints, Multiscale, InducingPatches
 from ..kernels import Kernel, SquaredExponential, Convolutional
 from .dispatch import Kuf
 
 
-@Kuf.register(InducingPoints, Kernel, object)
-def _Kuf(inducing_variable: InducingPoints, kernel: Kernel, Xnew: tf.Tensor):
+@Kuf.register(InducingVariables, Kernel, TensorLike)
+def Kuf_fallback(inducing_variable, kernel, X):
+    return Kuf(kernel, inducing_variable, X)
```

Comment:
I think we should not have this fallback, which allows parameter types to be switched. Allowing functions to be called with reversed parameters presents an inconsistent interface.